src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -335,6 +336,98 @@ export type TypeContext = {
     inJsonNamespace: boolean;
 };
 
+interface StringType {
+    getType(): string;
+    getConstType(): string;
+    getSMatch(): string;
+    getRegex(): string;
+    createStringLiteral(inner: Sourcelike): Sourcelike;
+    wrapToString(inner: Sourcelike): Sourcelike;
+    wrapEncodingChange(
+        qualifier: Sourcelike[],
+        fromType: Sourcelike,
+        toType: Sourcelike,
+        inner: Sourcelike
+    ): Sourcelike;
+    emitHelperFunctions(): void;
+}
+
+export function addQualifier(qualifier: Sourcelike, qualified: Sourcelike[]): Sourcelike[] {
```

Comment:
None of these new functions/interfaces/classes should be `export`ed, unless I'm missing something.